### ISSUE TITLE
Sentry messages should state the veteran rank name (not number) of the unit spotted

### ISSUE THREAD

**Issue by hugomflavio** [longturn/freeciv21#2369]
**Describe the bug**
battle messages display the veteran rank name of the units involved (e.g.  "disciplined Bowmen"), but sentry units display the veteran rank number ("v3 Bowmen"). 

**To Reproduce**
1. Sentry a unit
2. Have a veteran unit move within the sentry-trigger area.

**Expected behavior**
Sentry messages should display the veteran rank name.

**Screenshots**
![image](https://github.com/user-attachments/assets/8e0116e1-37cd-441e-a2d9-ec412449b3e4)

**Platform and version (please complete the following information):**
 - OS: Linux
 - Freeciv21 version: e.g. 3.1-beta.1

**Comment by jwrober** [comment]
Maybe both? Your units at [tile] were bombarded by a [nation] [vet level name] ([vet level number]) [unit]. 

Your units at (27, 17) were bombarded by a Otomi disciplined (v2) Bowmen [id:167].

**Comment by hugomflavio** [comment]
Yep that's a good idea. Especially since rank names may change from server to server

**Comment by jwrober** [comment]
I think it is too, good idea. onestly both are good - e.g (v3 150%)

**Comment by XHawk87** [comment]
Do we want to make `disciplined (120%) Bowmen` the standard way to display units in the messages? If so, it might make sense to have a function to build that string, and use it everywhere it's needed.
